academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+
+* The chains use relayers to communicate;
+* Chains can use different relayers to send messages to each other;
+* Each side of the relay uses the light client of the other chain to quickly verify incoming messages.
+
+In fact, the relayer listens and picks up messages from a channel to forward them. Both chains implement IBC and first initiate a connection to then establish channels. Notice that relayers ensure that both chains know that a package has been transmitted.
+
+![IBC module](images/ibcmodule.png)
+
+The interface exposed by the standard IBC implementation is referred to as the [IBC handler](https://github.com/cosmos/ibc/tree/master/spec/core/ics-025-handler-interface). Modules can use the IBC handler for client, connection, and channel lifecycle management, and to send and receive packets.
+
+![IBC handler](images/ibchandler.png)
+
+For each connection between two blockchains, different channels can be established. **Channels** provide the ordered exactly-once delivery of packets and handle module permissioning through **ports**. A **module** is bound to a port and channels are end-to-end between two ports. The relevant abstraction layer for IBC application developers is that of channels and ports. IBC modules can bind to any number of ports.
+
+IBC clients are **light clients** that are identified by a unique client ID. IBC clients track the consensus state of other blockchains and the proof specs of those blockchains required to properly verify proofs against the client's consensus state. A client can be associated with any number of connections to the counterparty chain.
+
+## Implemantations
+
+IBC is not just a specification on a whitepaper, it is used for hundreds of thousands of transactions per day across various chains.
+
+<HighlightBox type="tip">
+
+Have a look at the [Map of Zones](https://mapofzones.com/?testnet=false&period=24&tableOrderBy=totalIbcTxs&tableOrderSort=desc) to get an overview of blockchains with IBC implementation. In it, you can see some blockchains like the Cosmos Hub or Osmosis connected to a lot of other blockchains - they are used by many applications.
+
+</HighlightBox>
+
+Some **IBC-related implementations** to highlight are:
+
+* **The [IBC module](https://github.com/cosmos/ibc-go):** The implementation of the IBC protocol for the Cosmos SDK;
+* **[Hermes](https://hermes.informal.systems/relayer.html):** A widely-used relayer written in Rust;
+* **The [Gravity Bridge](https://github.com/cosmos/gravity-bridge):** A great example of a peg-zone, which allows the transfer of ERC-20 tokens to Cosmos-based blockchains and back.
+
+Some **IBC clients** described in the ICS and implemented in the IBC module are:
+
+* **`solomachine` light client:** Devices such as phones, browsers, or laptops;
+* **Tendermint light client:** The default for Cosmos SDK chains;
+* **Localhost (loopback) client:** Useful for testing, simulation, and relaying packets to modules on the same application.
```

Comment:
This one would best not be mentioned, instead we could link the ibc website to check the status of light client development.
https://ibcprotocol.org/lightClients/